ts/components/staking/wizard/wizard_flow.tsx

```diff
@@ -336,7 +336,7 @@ export const RecommendedPoolsStakeInputPane = (props: StakingInputPaneProps) =>
                                 key={rec.pool.poolId}
                                 name={stakingUtils.getPoolDisplayName(rec.pool)}
                                 collectedFees={rec.pool.currentEpochStats.totalProtocolFeesGeneratedInEth}
-                                rewards={1 - rec.pool.nextEpochStats.approximateStakeRatio}
```

Comment:
This was a 🐛 right?

Reply:
yes! https://app.asana.com/0/1153534547690690/1155895391339009 thank you!!

Reply:
(I think staked is also incorrect, but I can look at that in a separate PR)

Reply:
I'll fix that no worries. It seems like my task was a duplicate of that one 😄